web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc

```diff
@@ -8,6 +8,12 @@ author = "Thomas Fischer"
     parent = "workflows"
 +++
 
+## Unsupported Processes
+So far, HydroMechanics (including that with LIE) and ThermoHydroMechanics, which
+ use linear element for thermal and hydraulic processes and quadratic element
```

Comment:
```suggestion
 use linear elements for thermal and hydraulic processes and quadratic elements
```

Reply:
Thanks. I just changed the way of the expression. Please take  a look again.